conans/client/cmd/search_v2.py

```diff
@@ -0,0 +1,31 @@
+import fnmatch
+from collections import OrderedDict, namedtuple
+
+from conans.errors import NoRemoteAvailable
+from conans.search.search import (search_recipes)
+
+
+class Search(object):
```

Comment:
An object, without state, is probably better a function, you can pass the ``app`` as argument, it will have the collaborators inside.